src/main/java/se/audit/aop/AuditAspect.java

```diff
@@ -46,6 +46,7 @@ public void auditBeforeMessagesServerMethod(JoinPoint joinPoint) {
             returning = "returnValue")
     public Object auditAfterMessageServerMethod(JoinPoint joinPoint, Object returnValue) {
         String returnValueAsString = returnValue == null ? "null" : truncateString(returnValue.toString());
+        System.out.println("du kommentar som sonar borde hitta");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)